### PR TITLE
Scylla Manager docker integration

### PR DESCRIPTION
**Pull request by ajankovic**
Added ability to allow running Scylla Manager in a docker container.

Still working on full compatibility.

### CODE REVIEW

File: ccmlib/cluster_factory.py [hunk 1]
```diff
@@ -2,15 +2,16 @@
...
 import yaml
 
-from ccmlib import common, repository
+from ccmlib import common
+from ccmlib import repository
 from ccmlib.cluster import Cluster
 from ccmlib.dse_cluster import DseCluster
 from ccmlib.scylla_cluster import ScyllaCluster
-from ccmlib import repository
+from ccliib.docker_cluster import DockerCluster
```
Comment (fgelcer) [inline ccmlib/cluster_factory.py:10]: there is already the ScyllaDockerCluster in ccmlib

File: setup.py [hunk 2]
```diff
@@ -24,7 +24,7 @@
     url='https://github.com/pcmanus/ccm',
     packages=['ccmlib', 'ccmlib.cmds'],
     scripts=[ccmscript],
-    install_requires=['pyYaml', 'psutil', 'six >=1.4.1', 'requests'],
+    install_requires=['pyYaml', 'psutil', 'six >=1.4.1', 'requests', 'docker'],
```
Comment (fgelcer) [inline setup.py:27]: this was already addressed in #263 or if this one will be merged first... chat with @Orenef11 to avoid duplicate work

**Review by fgelcer — COMMENTED** [review]
Few comments below

File: ccmlib/cluster_factory.py [hunk 3]
```diff
@@ -2,15 +2,16 @@
 
 import yaml
 
-from ccmlib import common, repository
+from ccmlib import common
+from ccmlib import repository
 from ccmlib.cluster import Cluster
 from ccmlib.dse_cluster import DseCluster
 from ccmlib.scylla_cluster import ScyllaCluster
-from ccmlib import repository
```
[the remark this answers is not in the document]
Comment (fruch) [reply ccmlib/cluster_factory.py:9]: it just move a few lines up

File: ccmlib/cluster_factory.py (hunk 1 above)
Comment (fruch) [reply ccmlib/cluster_factory.py:10]: this line look like a typo anyhow, `ccliib`

**Comment by fruch** [comment]
we are not using the docker backend of ccm at all for now, so no point of keep this PR open...

**Pull request closed without merging**